trunk/problems/POISSON/GALERKIN/my_paraview_driver.F90

```diff
@@ -98,6 +113,12 @@ subroutine my_paraview_driver(IParAttr)
    enddo
 !
   90 continue
+
+!..deallocation only if using VTU output format
+   if(VIS_VTU) then
+      deallocate(IPARATTR_VTU)
+      deallocate(ELEM_TYPES)
```

Comment:
@ac1512 I don't think it's a good idea to have `ELEM_TYPES` deallocated by the user in a driver file that doesn't do the allocation of this array. If `geom2vtk` isn't called before (e.g. user not writing geometry data in each time step), then this crashes here.